packages-internal/test-utils/src/index.ts

```diff
@@ -15,9 +15,12 @@ export {} from './initMatchers';
 export * as fireDiscreteEvent from './fireDiscreteEvent';
 export * as userEvent from './userEvent';
 export { default as flushMicrotasks } from './flushMicrotasks';
+export { default as reactMajor } from './reactMajor';
```

Comment:
`reactMajor` is unused.

```suggestion
```

Reply:
It's used 🤔 for example: https://github.com/mui/material-ui/pull/43155/files#diff-66778813531b312efe75f8f19f05bc15abfc12cb31c8fa01e4b250fd172cc283R9